opencti-platform/opencti-graphql/src/modules/attributes/internalObject-registrationAttributes.ts

```diff
@@ -101,6 +101,7 @@ const internalObjectsAttributes: { [k: string]: Array<AttributeDefinition> } = {
     { name: 'default_time_field', type: 'string', mandatoryType: 'no', multiple: false, upsert: false },
     { name: 'account_status', type: 'string', mandatoryType: 'external', multiple: false, upsert: false },
     { name: 'account_lock_after_date', type: 'date', mandatoryType: 'no', multiple: false, upsert: false },
+    { name: 'administrated_organizations', type: 'string', mandatoryType: 'no', multiple: true, upsert: false },
```

Comment:
do we need to declare it as attribute ? since it's not stored, only get through graphql API

Reply:
I agree